crates/parcel_filesystem/src/in_memory_file_system.rs

```diff
@@ -16,29 +18,29 @@ enum InMemoryFileSystemEntry {
 /// In memory implementation of the `FileSystem` trait, for testing purpouses.
 #[derive(Debug)]
 pub struct InMemoryFileSystem {
-  files: RefCell<HashMap<PathBuf, InMemoryFileSystemEntry>>,
-  current_working_directory: RefCell<PathBuf>,
+  files: Arc<RwLock<HashMap<PathBuf, InMemoryFileSystemEntry>>>,
```

Comment:
There's no need to force send/sync over this; you could implement a lock based FS that wraps any possible implementation for example and implements Send/Sync;

Reply:
Fixed